samples/contractupdate/src/Root.tsx

```diff
@@ -1,51 +1,21 @@
 import React, { useEffect, useState } from 'react';
 import { Alert, Col, Container, Row } from 'react-bootstrap';
-import { Network, withJsonRpcClient } from '@concordium/wallet-connectors';
-import { SignClientTypes } from '@walletconnect/types';
+import { withJsonRpcClient } from '@concordium/wallet-connectors';
```

Comment:
I'm guessing this reference should be moved to `@concordium/react-components` then, since theres no explicit dependency on the module anymore?

Reply:
Indeed! There were quite a few of these. Fixed them all.